scripts/exglobal_archive.py

```diff
@@ -31,16 +31,24 @@ def main():
             'restart_interval_gdas', 'restart_interval_gfs',
             'AERO_ANL_RUN', 'AERO_FCST_RUN', 'DOIBP_WAV', 'DO_JEDIOCNVAR',
             'NMEM_ENS', 'DO_JEDIATMVAR', 'DO_VRFY_OCEANDA', 'FHMAX_FITS',
-            'IAUFHRS', 'DO_FIT2OBS']
+            'IAUFHRS', 'DO_FIT2OBS', 'NET']
 
     archive_dict = AttrDict()
     for key in keys:
-        archive_dict[key] = archive.task_config[key]
+        try:
+            archive_dict[key] = archive.task_config[key]
+        except KeyError:
+            archive_dict[key] = archive.task_config.get(key)
+            print(f"Key ({key}) not found in task_config")
```

Comment:
Why not just use `get()` from the jump?
```suggestion
        archive_dict[key] = archive.task_config.get(key)
        if archive_dict[key] is None:
            print(f"Key ({key}) not found in task_config")
```